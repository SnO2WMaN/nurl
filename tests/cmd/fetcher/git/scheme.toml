args = [
    "git://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git",
    "v1.9.0",
    "--fetcher",
    "fetchgit",
]
