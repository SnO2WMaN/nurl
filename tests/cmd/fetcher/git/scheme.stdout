fetchgit {
  url = "git://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git";
  rev = "v1.9.0";
  hash = "sha256-W38CWoPnkt0+VkVp+uW+sjWjo6MXYq2eTXRxWKyQph8=";
}